experiments/convert_traj.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+"""
+Convert trajectories from `imitation` format to openai/baselines GAIL format.
+"""
+
+import argparse
+import os
+from pathlib import Path
+import pickle
+from typing import List
+
+import numpy as np
+
+from imitation.util import rollout
+
+
+def convert_trajs_to_sb(trajs: List[rollout.Trajectory]) -> dict:
+  """Converts Trajectories into the dict format used by Stable Baselines GAIL.
+  """
+  trans = rollout.flatten_trajectories(trajs)
+  return dict(
+    acs=trans.acts,
+    rews=trans.rews,
+    obs=trans.obs,
+    ep_rets=np.array([np.sum(t.rews) for t in trajs]),
+  )
+
+
+def main():
```

Comment:
This is indented 4-spaces whereas the rest of the file is 2-spaces. I think we are using 2-spaces for this project?